app-backend/batch/src/main/scala/stacExport/WriteStacCatalog.scala

```diff
@@ -286,7 +286,7 @@ final case class WriteStacCatalog(exportId: UUID)(
      */
     val catalogBuilder =
       new StacCatalogBuilder[StacCatalogBuilder.CatalogBuilder.EmptyCatalog]()
-    val stacVersion = "0.7.0"
+    val stacVersion = "0.8.0-rc1"
```

Comment:
`stac_validator` can't find this version (or with a `v` in front of it) which is a little bit frustrating, but probably fine :tm: since we can obviously see the tag in the spec repo